### ISSUE TITLE
I think Rwallet should support lightning on custodial way using Lndhub

### ISSUE THREAD

**Issue by bitcoinuser** [rsksmart/rwallet#365]
Hi,

I don't think Rwallet should implement a non custodial lightning wallet because it's complex deal with channels..... 
Instead I think Rwallet should implement https://lndhub.io/about so I could link wallets like lntxbot or even Bluewallet, exporting lndhub data and importing it on Rwallet.

1- I think Rwallet when on Add wallet window, on import existing Wallet could allow import not only seed but data from Lndhub. 
2- When the user import a lndhub wallet, should show the main Rwallet window but without option to add assets.

**Comment by bitcoinuser** [comment]
Hi @chrisli30. If someone create a PR to this feature, would you accept on Rwallet? Thanks.

**Comment by chrisli30** [comment]
Do you mean just by monitoring the address for an external wallet?

We already have that in the pipeline and will deliver in a couple of weeks 😄

**Comment by bitcoinuser** [comment]
> Do you mean just by monitoring the address for an external wallet?
> 
> We already have that in the pipeline and will deliver in a couple of weeks

No, it's not this. It's like import a "private key" that give full access to a lightning node. So I will be able to receive and send bitcoins on lightning using rwallet directly.

**Comment by chrisli30** [comment]
> > Do you mean just by monitoring the address for an external wallet?
> > We already have that in the pipeline and will deliver in a couple of weeks
> 
> No, it's not this. It's like import a "private key" lndhub that gives full access to a lightning node. So I will be able to receive and send bitcoins on lightning using rwallet directly.

Got it. I like it, yes if you can create a PR for this feature we would like to include it. It will be an addition to our address monitoring feature and become the 4th type of wallet.

**Comment by cryptosalomao** [comment]
Hello, everyone. I talked to @bitcoinuser privately, and i already started working on this. I'll be posting my progress here to discuss fixes or improvements of this proposal. I'll make a PR anytime soon. I hope it gets accepted.

**Comment by chrisli30** [comment]
@cryptosalomao Awesome, are you guys based in Brazil?

**Comment by cryptosalomao** [comment]
@chrisli30 yes, we are. I already finished the lndhub integration, and now i'm finishing lndconnect stuff. I also have started to code the pages (plug, overview, send/receive).

**Comment by bitcoinuser** [comment]
Hi @chrisli30 and @cryptosalomao.

1- I think on Add wallet, the 4th wallet should have the name:

Connect a Lightning Wallet

And the description:

Connect to a Third Party Service to access a Lightning Wallet.

2- After clicking on Connect to a Lightning Wallet, will appear a window like the one of Import Wallet, where the will be able to paste or read a qrcode with LNDhub or Lnd Connect.

The title will be:

Connect Lightning Wallet

The description will be:

Enter the credential that gives access to a Lightning Wallet. Supported formats: LNDHub and Lndconnect.

**Comment by bitcoinuser** [comment]
Thinking better, maybe instead of (Connect Lightning Wallet), should be (Access Lightning Wallet)
